examples/tutorials/code_1_python_api.py

```diff
@@ -52,13 +52,13 @@
 # a ``ValueError`` will be raised. At least one of the results must have the type ``objective``,
 # the metric that is minimized by the algorithm.
 
+if __name__ == '__main__':
```

Comment:
This one is part of the sphink-gallery, which automatically convert the script into a sphinx page and a jupiter notebook. Maybe there could be a warning for macos user to try it using the jupiter notebook or encapsulating the code in a main function and running under `if __name__ == '__main__'`.